api/v1/convert/convert.go

```diff
@@ -107,6 +107,7 @@ func NVDVulns(nvdVulns []*nvdtoolscache.NVDCVEItemWithFixedIn) ([]*v1.Vulnerabil
 			Link:        nvd.Link(vuln.CVE.CVEDataMeta.ID),
 			MetadataV2:  Metadata(m),
 			FixedBy:     vuln.FixedIn,
+			Severity:    string(DatabaseSeverityToSeverity(m.GetDatabaseSeverity())),
```

Comment:
why is `Severity` a string field and not enum?

Reply:
not sure I think the gRPC API predates my StackRox tenure